docs/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM vault.habana.ai/gaudi-docker/1.4.0/ubuntu20.04/habanalabs/pytorch-installer-1.10.2:1.4.0-442
+
+# Need node to build doc HTML. Taken from https://stackoverflow.com/a/67491580
+RUN apt-get update && apt-get install -y \
```

Comment:
Not sure if this is the simplest way to install `node` in this container - it's at least the one that works :)